src/posthog-core.ts

```diff
@@ -1038,6 +1064,11 @@ export class PostHog {
      * practice to call identify on the original, anonymous ID
      * right after you've aliased it.
      *
+     * A user can be initalised as identified by segment or posthog boostrap
```

Comment:
this is for `if (!this.get_property('$device_id')) {` right? could you move it to the code there then?